apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx

```diff
@@ -34,24 +40,22 @@ export const JudgeOverview: React.FC = () => {
     setCourtCaseNumberErrorMessage,
   ] = useState('')
   const [workingCase, setWorkingCase] = useWorkingCase()
-  const uContext = useContext(userContext)
 
   useEffect(() => {
     document.title = 'Yfirlit kröfu - Réttarvörslugátt'
   }, [])
 
+  const { data } = useQuery(CaseQuery, { variables: { id: id } })
```

Comment:
Queries and mutations should follow the `Input objects naming conventions`: https://github.com/island-is/handbook/blob/master/docs/api-design-guide/graphql-naming-conventions.md#input-objects-naming-conventions

Reply:
I was not aware of that. Will modify the code accordingly.

Reply:
I think I have addressed all naming issues.